plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -81,6 +78,13 @@ public BigQueryMetadata(BigQueryClient bigQueryClient, BigQueryConfig config)
     public List<String> listSchemaNames(ConnectorSession session)
     {
         log.debug("listSchemaNames(session=%s)", session);
+        return listRemoteSchemaNames(session).stream()
+                .map(schema -> schema.toLowerCase(ENGLISH))
```

Comment:
is this needed? we can assume it's engine's job

Reply:
There are tests missing for mixed case datasets. I'll add those and see what happens on removing. AFAIR I added it there explicitly.

Reply:
I am not saying "it musn't be here". I just pointed out it's not at all obvious why this is here, so a comment is due

Reply:
The engine doesn't do anything with the return value of this method. The existence of a schema is checked by directly running `listsSchemaNames(session).contains(schemaName)`.

Also, all existing ConnectorMetadata impls perform similar normalisation. I think a comment should be there on `ConnectorMetadata#listSchemaNames` instead.